src/status_im/config.cljs

```diff
@@ -167,5 +167,5 @@
 
 (def default-kdf-iterations 3200)
 
-(def community-accounts-selection-enabled? true)
+(def community-accounts-selection-enabled? (enabled? (get-config :ACCOUNT_SELECTION_ENABLED "0")))
```

Comment:
not sure why this is here but we should move to feature flags :)

Reply:
It's here because it was created before you created the new feature flags solution. Should be moved definitely.